torch/_tensor_docs.py

```diff
@@ -1785,6 +1785,49 @@ def callable(a, b) -> number
             [ 0.0000,  0.0000,  0.0000,  1.2300]])
 """)
 
+add_docstr_all('scatter_add_',
+               r"""
+scatter_add_(dim, index, other) -> Tensor
+
+Adds all values from the tensor :attr:`other` into :attr:`self` at the indices
+specified in the :attr:`index` tensor in a similar fashion as
+:meth:`~torch.Tensor.scatter_`. For each value in :attr:`other`, it is added to
+an index is specified by its index in :attr:`other` for dimension != :attr:`dim`
```

Comment:
nit: ... index is specified ... --> ... index which is specified ...
nit?: != -->:math:`\neq`